main.py

```
@@ -111,87 +112,116 @@ async def main():
+    # ------------------------------------------------------------------ #
+    # Main navigation loop — allows returning to the menu from anywhere.  #
+    # ------------------------------------------------------------------ #
+    while result != "quit":
+
+        # --- Start menu ---
+        if result == "menu":
+            while True:
+                menu_screen = StartMenu(screen)
+                result = await menu_screen.run()
+                if result == "credits":
+                    credits = CreditsScreen(screen)
```

Ruff (0.15.9)

[error] Variable `credits` is shadowing a Python builtin

(A001)